docs/features/feature-guides/logical-models/centralized-management.md

```diff
@@ -0,0 +1,68 @@
+# Centralized Management
+
+:::info
+This feature is currently in private beta in DataHub Cloud. Reach out to your DataHub Cloud representative to learn more.
+:::
+
+## How It Works
+
+Centralized Management allows governance of all physical children at the logical parent level. This means that metadata changes made to the logical model will be propagated to all its physical children.
```

Comment:
We should include a specifc description of change types which are copied from parent to child here! So people can clearly understand the scenarios we've tested to work vs. edge cases.